### ISSUE TITLE
Add support for all major browsers

### ISSUE THREAD

**Issue by ghost** [OPERANDOH2020/PlusPrivacy#58]
I think It would be great to support at least Firefox to give the users the freedom to choose to use a privacy-friendly browser.

Although Edge, Safari and other browsers supports it's important too.

I wait for news on this, regards.

**Comment by jeroenev** [comment]
and at least not block other browsers from logging into the website, i understand the extension only works for chrome so you can't change privacy settings on firefox, but i should at least be able to create aliasses for mails on firefox

**Comment by rugk** [comment]
Indeed, at least support for Firefox should be done. And it should be quite easy, based on the WebExtension architecture…

**Comment by Mastaleru** [comment]
The Firefox add-on was already added: https://addons.mozilla.org/en-US/firefox/addon/plusprivacy/.
The new version is not including ABP because users use their favorite adbocker: 
https://github.com/OPERANDOH2020/PlusPrivacy/tree/cross-platform/clients/browser-ext.
The Firefox version still have some small UI problems, but them will be fixed in a week from now.
Thank you
